packages/models/src/lib/global-options.ts

```diff
@@ -12,6 +12,7 @@ export const globalOptionsSchema = z.object({
       description: 'Outputs additional information for a run',
     })
     .default(false),
+  // @TODO move to cli package as it is only used there
```

Comment:
Good point, probably doesn't need to be a schema then.

Reply:
In another PR